NAMESPACE

```diff
@@ -28,3 +28,5 @@ export(.RCodeInOptionsIsOk)
 S3method(.RCodeInOptionsIsOk, list)
 S3method(.RCodeInOptionsIsOk, default)
 export(.parseAndStoreFormulaOptions)
+export(substituteInteractionSymbol)
+export(interactionSymbol)
```

Comment:
I think given the low relative importance of this functionality, just the export of the substitute function should be enough.

Reply:
Well, there are already quite a few `paste(strings, collapse = "\u2009\u273b\u2009")` in there (I count 5 analyses using it, 6 if logistic regression would actually use the nice symbol instead of `x`). Since we don't encode the factor levels in the data set (although maybe we should), the replacement function is kinda risky.

Reply:
Hmm if it's used like that I guess it'd be nice to export. Just make sure the naming of the two exports match (as per the next comment)

`gsubInteractionSym(paste(strings, collapse = ":"))` seems a bit over the top :p

Reply:
> Since we don't encode the factor levels in the data set (although maybe we should), the replacement function is kinda risky.

If we don't trust people to create sensible columnnames we can also not trust them to make sensible levels right?

Reply:
Kinda glossed over that, why don't we encode the levels?

Reply:
there was no reason to encode them... so far :P

Reply:
but when separating the column names from the factor levels in the R output (see example above) it could very well be that people use some weird symbols that mess up my regex.

Reply:
So we should make two new issues I think, one requesting that levels are encoded along with the columnnames.
The other I was talking about with Don and that would be to also include those encodings in jaspTools so that the unittests test the exact same way as JASP.

Reply:
Here you go: https://github.com/jasp-stats/INTERNAL-jasp/issues/1171, https://github.com/jasp-stats/INTERNAL-jasp/issues/1172